src/isomorphic/classic/element/ReactElementType.js

```diff
@@ -23,7 +23,7 @@ export type ReactElement = {
   $$typeof: any,
   type: any,
   key: any,
-  ref: any,
+  ref: string | (elem: ?HTMLElement) => void,
```

Comment:
Is there any way flow coverage could be integrated into flow CLI? Many people don't use Nuclide and when they try flow, they aren't aware flow coverage exists.